code/__HELPERS/text.dm

```diff
@@ -8,6 +8,13 @@
  *			Misc
  */
 
+// Run all strings to be used in an SQL query through this proc first to properly escape out injection attempts.
+/proc/sanitizeSQL(t as text)
+	return html_encode(t);
+
+/proc/unsanitizeSQL(t as text)
+	return html_decode(t);
```

Comment:
This is the worst way to sanitize SQL.

The DM DBCon even have a `dbcon.Quote()` thing to properly quote strings. This is just going to fuck up anything that uses special chars and make it worse.

Also semicolons in DM 🤮